OWNERS

```diff
@@ -1,15 +1,8 @@
 approvers:
-  - abhi-g
   - animeshsingh
-  - ashahba
   - Bobgy
-  - gabrielwen
-  - hougangliu
-  - IronPan
   - jlewi
-  - kkasravi
   - kunmingg
-  - lluunn
   - richardsliu
```

Comment:
Please remove @kunmingg and @richardsliu as well.

Reply:
Done